templates/intersection.xml.em

```diff
@@ -853,15 +853,46 @@ if crosswalk or stopline:
         </connection>
     </junction>
 </OpenDRIVE>
-@[if crosswalk or stopline]
-<!-- GeoJSON information for the crosswalks
+<!-- GeoJSON information
 Generated GeoJSON can be verified using: https://geojson.io
 @{
 }@
 
 {
+  "_comment": "Features of type 'hasOncomingTraffic' are temporary hints and ignored by Maliput.",
   "features": [
-@[if crosswalk]
+    {
+      "properties": {
+        "Id": "9_0_1",
+        "Type": "Lane",
+        "HasOncomingTraffic": true
+      },
+      "type": "Feature"
+    },
```

Comment:
I checked this in `geojson.io` and it complains about that geometry information is missing. @liangfok

Reply:
It seems that each feature needs to have a geometry and property information

Reply:
We could add some dummy geometry:

```
    {
      "geometry": {
        "coordinates": [
          0, 0
        ],
        "type": "Point"
      },
      "properties": {
        "Id": "9_0_1",
        "Type": "Lane",
        "HasOncomingTraffic": true
      },
      "type": "Feature"
    },
```
Here we can make the point to indicate the start point of the lane

Reply:
Okay, adding a dummy geometry is fine. Can we include a comment informing future viewers that it is just a dummy, temporary, and currently not used by Maliput?

Reply:
Thanks, I suppose it's not really convenient to only add these has-oncoming-traffic hints to only one map (resources/intersection/intersection_3_3m_width_6m_radius_stopline.xodr)? That's fine as long as the hints actually apply to all of the maps.

Reply:
Correct, I added it to the template, meaning all the intersection maps have this extra information as well.